alacritty/src/renderer/text/glyph_cache.rs

```diff
@@ -264,19 +264,22 @@ impl GlyphCache {
         loader.load_glyph(&glyph)
     }
 
-    /// Clear currently cached data in both GL and the registry.
-    pub fn clear_glyph_cache<L: LoadGlyph>(&mut self, loader: &mut L) {
+    /// Reset currently cached data in both GL and the registry to default state.
+    pub fn reset_glyph_cache<L: LoadGlyph>(&mut self, loader: &mut L) {
         loader.clear();
-        self.cache = HashMap::default();
+        self.cache = Default::default();
 
         self.load_common_glyphs(loader);
     }
 
-    pub fn update_font_size<L: LoadGlyph>(
+    /// Update the inner font size.
+    ///
+    /// NOTE: To reload the renderers's fonts [[`Self::reset_glyph_cache`]] should be called
```

Comment:
```suggestion
    /// NOTE: To reload the renderers's fonts [`Self::reset_glyph_cache`] should be called
```

I'd assume the double brackets were just a typo? Or is there actually any formatting difference in the output with double brackets?